Thesis.tex

```diff
@@ -222,11 +225,85 @@ \subsection{Reasonable and non-discriminatory licensing}
  
 Another important case related to RAND is Microsoft versus Motorola. In 2012, the United States Court of Appeals for the Ninth Circuit ruled that a patent holder's agreement to a policy like RAND creates a legally binding contract \parencite{MicrosoftVsMotorola}.
 
+\chapter{Discourse}
+\label{ch:Disc}
+
+\section{Copyright in the context of the arts}
+\label{sec:CopyArts}
+
+\subsection{Pro IP}
+
+\subsection{Against IP}
+There are several possible problems with copyrights in the context of the fine arts that come to the mind of the authors of this thesis. 
+
+Let us first look into the area of music.
+\begin{itemize}
+	\item There are 12 tones in the chromatic scale, and normal scales consist of 8 tones. If we consider that these have been used for hundreds of years, there is a limitation of the number of possible arrangements in a certain cycle. This means that - whatever a composer does - there is a high probability that parts of the composition exist in parts of other compositions. This does not mean that no original compositions are possible anymore, but that it is quite certain that there are overlaps with other musical pieces. If one considers that the teachers in the craft use similiar teaching techniques (as e.g. using variations of the scales for guitar solos) as well as the fact that we hear music without being consciously aware of it rather often, e.g. in shopping malls or passing someone using a radio, the possibility of unintentional inspiration is high. 
+	\item In the current age, music is often not the creation of an inspired artist but of composers creating goods based on market research, e.g. the current music taste and characterizations of idols for pre-teen and teenage girls, which does lead to floods of one-hit wonders and good singers that get burnt out for the financial gain of industry giants. As such one-hit wonders generate big revenue streams for the publishers that have also controlled the "compositions" (and I use that word loosely) and only have to pay for the actual performance and minor wages for the singers and musicians, the market gets saturated by low quality music leaving less space for really talented artists and making it hard for them to get acceptable contracts, as the risk for the publishers might be higher and the part of the revenue going to the artist is (rightfully) higher, hurting the industry's bottom line. This, however, leads to good artists giving up or not being able to generate an income by touring, as their music is not known enough, basically being drowned by low-quality noise.
```

Comment:
So what if the market is flooded with low quality music?
Or said differently: How does this relate to copyright?